roboteam_ai/src/coach/PassCoach.cpp

```diff
@@ -59,6 +59,7 @@ int PassCoach::determineReceiver(int passerID) {
     int bestRobotID = -1;
     for(auto &robot : world::world->getUs()) {
         if (robot.id == robotDealer::RobotDealer::getKeeperID() || robot.id == passerID) continue;
```

Comment:
this can probably put into a better if else without continues (continues can give weird behaviour)

Reply:
continue is fine